module-4/backend/week_two.md

```diff
@@ -4,9 +4,35 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. What is Webpack and why is it useful?
-2. When do you want to use event delegation?
+1. What is Webpack and why is it useful? 
+  - A frontend frame work to allow a platform for a server, and webtessting. Its useful because you can then update the application with live changes 
```

Comment:
not quite. Webpack is module bundler - it puts all of your assets, including Javascript, images, fonts, and CSS into one place - similar to the asset pipeline in Rails